spec/repo_spec.rb

```diff
@@ -14,20 +14,20 @@
     VCR.eject_cassette
   end
 
-  describe 'Get all repositories of a developer' do
-    it '(HAPPY) should find all repositories with valid group ID' do
-      get "api/v0.1/dev/#{Developer.first.username}/repos"
+  describe 'Get a repository of a developer' do
+    it '(HAPPY) should find a repository from an owner and a repository name' do
```

Comment:
this test is good but can you do it with a repo that we do not have in the database